library/error.c

```diff
@@ -821,6 +825,11 @@ void mbedtls_strerror( int ret, char *buf, size_t buflen )
         mbedtls_snprintf( buf, buflen, "PADLOCK - Input data should be aligned" );
 #endif /* MBEDTLS_PADLOCK_C */
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( use_ret == -(MBEDTLS_ERR_PLATFORM_HW_ACCEL_FAILED) )
+        mbedtls_snprintf( buf, buflen, "PLATFORM - Hardware failed platform operation" );
```

Comment:
I appreciate you probably haven't changed the error text since the PR was first approved, but the error looks confusing to me. We haven't failed a platform operation - but some crypto operation. Could we change the phrasing to:

> PLATFORM - Hardware accelerator failed

It's vaguer, but truer to the intent of the error.

Reply:
OK